### PR TITLE
Rename `monitoring-grafana-mimir` page to `monitor-grafana-mimir`

### PR DESCRIPTION
**Pull request by jdbaldry**
#### Which issue(s) this PR fixes or relates to

Relates to #1663 

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/operators-guide/monitor-grafana-mimir/_index.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
   - Mimir dashboards
-aliases:
-  - visualizing-metrics/
+menuTitle: Monitor Mimir
+title: Monitor Grafana Mimir
+weight: 50
 ---
 
-# Monitoring Grafana Mimir
+# Monitor Mimir
```
Comment (osg-grafana) [inline docs/sources/operators-guide/monitor-grafana-mimir/_index.md:13]: I would leave this instance.

File: docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-menuTitle: "Config"
-description: "View an example Config dashboard."
-weight: 50
 aliases:
-  - ../../visualizing-metrics/dashboards/config/
+  - /../../visualizing-metrics/dashboards/config/
+  - /docs/mimir/latest/operators-guide/monitoring-grafana-mimir/dashboards/config/
+description: View an example Config dashboard.
+menuTitle: Config
+title: Grafana Mimir Config dashboard
```
Comment (osg-grafana) [inline docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md:7]:
```suggestion
title: Grafana Mimir configuration dashboard
```

File: docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
-title: "Grafana Mimir Config dashboard"
-menuTitle: "Config"
-description: "View an example Config dashboard."
-weight: 50
 aliases:
-  - ../../visualizing-metrics/dashboards/config/
+  - /../../visualizing-metrics/dashboards/config/
+  - /docs/mimir/latest/operators-guide/monitoring-grafana-mimir/dashboards/config/
+description: View an example Config dashboard.
+menuTitle: Config
```
Comment (osg-grafana) [inline docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md:6]:
```suggestion
menuTitle: Configuration
```

File: docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
+title: Grafana Mimir Config dashboard
+weight: 50
 ---
 
 # Grafana Mimir Config dashboard
```
Comment (osg-grafana) [inline docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md:11]:
```suggestion
# Grafana Mimir configuration dashboard
```

**Review by osg-grafana — APPROVED** [review]
Looks good from what I can tell.

File: docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md (hunk 2 above)
Comment (jdbaldry) [reply docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md:7]: Unrelated to this change and I believe the title matches the dashboard title

File: docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md (hunk 4 above)
Comment (jdbaldry) [reply docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md:11]: Unrelated to this change and I believe the title matches the dashboard title

File: docs/sources/operators-guide/monitor-grafana-mimir/dashboards/alertmanager-resources/index.md [hunk 5]
```diff
@@ -1,10 +1,11 @@
 ---
-title: "Grafana Mimir Alertmanager resources dashboard"
-menuTitle: "Alertmanager resources"
-description: "View an example Alertmanager resources dashboard."
-weight: 20
 aliases:
-  - ../../visualizing-metrics/dashboards/alertmanager-resources/
+  - /../../visualizing-metrics/dashboards/alertmanager-resources/
```
Comment (pracucci) [inline docs/sources/operators-guide/monitor-grafana-mimir/dashboards/alertmanager-resources/index.md:3]: This edit looks wrong to me. It was relative, now it's absolute. I just manually tried, and doesn't work anymore.

_Same applies to all other places._

I open a PR to fix them.

Comment (osg-grafana) [reply docs/sources/operators-guide/monitor-grafana-mimir/dashboards/alertmanager-resources/index.md:3]: Please build the entire website for testing.

**Pull request merged**